tests/models.py

```diff
@@ -0,0 +1 @@
+# this file is necessary for some reason for test models such as tests/sentry/db/models/fields/bitfield/test_bitfield.py
```

Comment:
did you mean to delete this one?

Reply:
it says it's necessary for some reason 🤔

Reply:
yeah I added the comment so the next person that tries to do this isn't confused -- apparently this is a load-bearing empty file 🙃